js/ui/sessionMode.js

```diff
@@ -106,8 +106,8 @@ const _modes = {
         panel: {
             left: ['endlessButton', 'appIcons'],
             center: [],
-            right: ['a11y', 'keyboard', 'aggregateMenu', 'userMenu',
-                    'dateMenu', 'hotCorner']
+            right: ['dateMenu', 'a11y', 'keyboard', 'aggregateMenu', 'userMenu',
```

Comment:
I don't think this change is right. The clock should stay where it currently is.